lca-ai-stack/deployment/lca-ai-stack.yaml

```diff
@@ -894,7 +897,7 @@ Resources:
       Handler: lambda_function.handler
       Layers:
         - !Ref TranscriptEnrichmentPythonLayer
-      MemorySize: 5120
+      MemorySize: 512
```

Comment:
This memory reduction seems incorrect, and unrelated to hotfix.  Can you dbl check, and maybe remove this change?